src/commands/force/source/retrieve.ts

```diff
@@ -65,11 +66,6 @@ export class Retrieve extends SourceCommand {
       summary: messages.getMessage('flags.retrievetargetdir.summary'),
       exclusive: ['packagenames', 'sourcepath'],
     }),
-    apiversion: Flags.string({
```

Comment:
`api-version` was on the command, but another `api-version` was the one actually being used in the code below.  I *think* this was just to get a char alias the hard way, but that meant `api-version` was being ignored.